src/Services/SyncSpeakersService.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Services;
+
+use Blumilk\Meetup\Core\Models\Meetup;
+
+class SyncSpeakersService
+{
+    public function syncSpeakers(Meetup $meetup, ?array $speakers): void
+    {
+        $meetup->speakers()->sync($speakers);
+    }
+}
```

Comment:
Do you need service for this operation? I think you can use this `$meetup->speakers()->sync($speakers);` instead of create new object instance and run method.